ragelib/image_fetcher.py

```diff
@@ -22,22 +22,20 @@ def get_graph_screenshot(self, url, driver):
         self.logger.debug(f"Begun fetching url {url}")
         driver.get(url)
         
-        try:
-            WebDriverWait(driver, timeout=10).until(expected.alert_is_present())
+        wait.until(expected.visibility_of((By.ID, 'graph_title')))

```

Comment:
graph_title element become visible immediately after progress_img (but progress_img doesn't persist)
-by this point, if there is no alert then no time is lost. if there is an alert then it will definitely also be present by the time the graph title is loaded. Tested: https://info.citrite.net/display/~svcacct_ragebot/CP-38420+Test